### PR TITLE
Fixes custom apearance merge bug and pin move bug

### PR DESCRIPTION
**Pull request by BFH-ktt1**
This PR addresses #1272  and #1280

### CODE REVIEW

File: src/main/java/com/cburch/logisim/circuit/appear/CircuitAppearance.java [hunk 1]
```diff
@@ -277,7 +281,7 @@ public boolean isDefaultAppearance() {
   public List<CanvasObject> getCustomObjectsFromBottom() {
     return super.getObjectsFromBottom();
   }
-
+
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/circuit/appear/CircuitAppearance.java:284]: Avoid spurious white space

File: src/main/java/com/cburch/logisim/std/wiring/PinAttributes.java [hunk 2]
```diff
@@ -80,7 +80,7 @@ public boolean isToSave(Attribute<?> attr) {
     if (attr == ATTR_DUMMY) return (V) "nochange";
     return super.getValue(attr);
   }
-
+
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/std/wiring/PinAttributes.java:83]: Avoid spurious white space.

**Review by maehne — APPROVED** [review]
Looks good to me.

File: src/main/java/com/cburch/logisim/circuit/appear/CircuitAppearance.java (hunk 1 above)
Comment (BFH-ktt1) [reply src/main/java/com/cburch/logisim/circuit/appear/CircuitAppearance.java:284]: This is caused by eclipse....

**Pull request merged**